sql/parser/parse_pass.sql

```diff
@@ -119,9 +124,9 @@ SELECT /* value argument with dot */ :a.b FROM t
 SELECT /* positional argument */ ? FROM t#SELECT /* positional argument */ :v1 FROM t
 SELECT /* multiple positional arguments */ ?, ? FROM t#SELECT /* multiple positional arguments */ :v1, :v2 FROM t
 SELECT /* NULL */ NULL FROM t
-SELECT /* octal */ 010 FROM t
-SELECT /* hex */ 0xf0 FROM t
-SELECT /* hex caps */ 0xF0 FROM t
+SELECT /* octal */ 010 FROM t#SELECT /* octal */ 8 FROM t
```

Comment:
It's part of the vitess testing code. The `#` separates the input from the expected. If there is no `#`, the expected output is exactly the input. In this case, while we can parse octal numbers, we always output them as decimal.